<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>btnComback</class>
 <widget class="QMainWindow" name="btnComback">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_95">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>131</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../image/image1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>QUANG TRUNG ( NGUYỄN HUỆ )</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>+ Sơ Yếu Lí Lịch :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_96">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>181</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../image/z6613118047752_c3aa6d80e061af947cde9c39260f13d9.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>- Sinh Năm :  Quý Dậu 1753</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>- Năm mất : 16 tháng 9 năm 1792</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>- Tên khai sinh : Hồ Thơm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>- Quê quán: Làng Tây Sơn, phủ Quy Nhơn (nay thuộc huyện Tây Sơn, tỉnh Bình Định)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>- Thân phụ: Hồ Phi Phúc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>- Thân mẫu: Nguyễn Thị Đồng</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>- Anh em: Nguyễn Nhạc (anh), Nguyễn Lữ (em)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>+ Sự nghiệp và đóng góp nổi bật :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>571</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>- Lãnh đạo khởi nghĩa Tây Sơn: Cùng với hai anh em, Nguyễn Huệ lãnh đạo phong trào Tây Sơn
, lật đổ các tập đoàn phong kiến Trịnh – Nguyễn, chấm dứt tình trạng phân chia Đàng Trong – Đàng
 Ngoài kéo dài suốt 2 thế kỷ.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>- Đánh bại ngoại xâm: Năm 1785, ông chỉ huy quân Tây Sơn đánh tan 5 vạn quân Xiêm tại trận
 Rạch Gầm – Xoài Mút. Năm 1789, ông dẫn đại quân ra Bắc, đánh bại 29 vạn quân Thanh trong trận
 Ngọc Hồi – Đống Đa, giải phóng Thăng Long, bảo vệ nền độc lập dân tộc.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>390</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>- Cải cách đất nước: Quang Trung tiến hành nhiều cải cách về hành chính, giáo dục và quân sự. Ông
 chủ trương sử dụng chữ Nôm trong thi cử và văn bản hành chính, thay thế chữ Hán, nhằm phát triển
 văn hóa dân tộc. </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>450</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>- Chính sách tôn giáo: Ông thực hiện chính sách khoan dung tôn giáo, cho phép các tôn giáo cùng tồn 
tại, đồng thời cải tổ Phật giáo bằng cách sáp nhập các chùa nhỏ và yêu cầu tăng sĩ phải qua kỳ thi
 tuyển.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnComback_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Quay Lại</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
